.gitignore

```diff
@@ -1,3 +1,5 @@
 dist
 src/lib
-node_modules
\ No newline at end of file
+node_modules
+.ds_store
+package-lock.json
```

Comment:
There is some recommendation that we should version control package-lock.json. I am not convinced and also prefer to ignore it. Maybe this is a topic worth discussing.

Reply:
Yes in some cases package-lock.json is necessary, I don't think this is the case for this module